native/src/routes/JpalTracking.tsx

```diff
@@ -30,43 +32,94 @@ const DescriptionContainer = styled.TouchableOpacity`
   justify-content: space-between;
   padding: 15px 0 5px;
 `
-const Input = styled(TextInput)`
+const Input = styled(TextInput)<{ editable: boolean; error: boolean }>`
   padding: 15px;
   border-width: 1px;
   border-color: ${props => (props.editable ? props.theme.colors.themeColor : props.theme.colors.textDisabledColor)};
+  ${props => (props.error ? 'border-color: red;' : '')};
```

Comment:
Not working. Border is not red, although an error occured. See attached screenshot.

Reply:
Oke, only working when entering the jpal-settings-screen with a deep-link.